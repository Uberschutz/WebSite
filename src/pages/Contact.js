import React, { Component } from 'react';
import '../styles/bootstrap.css';
import '../styles/Contact.css';

class Contact extends Component {
    render() {
        return (
            <div className="form-align">
                <div> <br/>
                <h9 className="h9-font">
                    Qui sommes nous ? Que faisons nous ?
                </h9> <br/> <br/>
                    <div className="description-txt">
                        <h6 className="title-bold">
                            • Überschutz :
                        </h6>
                            <h7>
                                Nous sommes un groupe de 7 étudiants d'Epitech de la promo 2021. <br/>
                                Notre groupe a été créé en octobre 2018 (tout d'abord 3 personnes), lors de la Moonshot, puis s'est complété courant janvier 2019.
                            </h7> <br/> <br/>
                            <h6 className="title-bold">
                                • Notre but :
                            </h6>
                            <h7>
                                Nous voulons qu'Internet soit un lieu de partage, de détente et de recherche. <br/>
                                Beaucoup trop de personnes et d'enfants sont harcelés, arnaqués ou confrontés à du contenu inapproprié.<br/>
                                Ainsi, nous souhaitons guider les plus jeunes afin qu'ils apprennent à utiliser Internet de la meilleure manière possible pour eux selon l'éducation que leurs parents veulent leurs donner.
                            </h7> <br/> <br/>
                            <h6 className="title-bold">
                                • Notre solution :
                            </h6>
                            <h7>
                                Aujourd'hui, en tant que jeunes adultes, nous trouvons que certains contenus sont choquants sur Internet et que ce n'est pas assez sécurisé. <br/>
                                Donc qu'en est-il pour un enfant ? <br/>
                                C'est pour cela que nous proposons un logiciel capable de monitorer la navigation Internet. <br/>
                                De ce fait, les enfants pourront être avertis que leur recherche n'est pas adaptée pour eux selon leurs parents. <br/>
                                Afin de respecter la liberté de l'enfant, Überschutz n'est ni bloquant ni intrusif. <br/>
                                C'est à dire qu'en fonction du paramétrage que vous choisirez, le logiciel n'interviendra qu'en tant que superviseur. Il laissera l'enfant naviguer mais se permettera d'afficher un message lorsque le contenu du site visité risque d'être inapproprié. <br/>
                                En conséquence, les parents pourront visualiser un détail statistique sur l'environnement virutel dans lequel vit l'enfant pour arriver à une discussion si cela est nécessaire.
                            </h7> <br/>
                        <br/>
                    </div>
                </div>
                <Faq/>
                <Form/>
            </div>
        )
    }
}

class Form extends Component {
    render() {
        return (
            <div>
                <br/>
                <h9 className="h9-font">
                    Vous êtes intéressés par notre solution ?
                </h9><br/>
                <h11>
                    N'hésitez pas à vous tenir informés en remplissant ce formulaire !
                </h11><br/>
                <h11>
                    Ou en nous contactant à l'adresse suivante : <h11 className="address">
                    uberschutz_2021@labeip.epitech.eu</h11>
                </h11>
                <form><br/>
                <div className="form-group">
                    <div className="form-group form-align">
                        <label>Name or Surname</label>
                        <input type="text" className="form-control form-box" placeholder="Name or Surname"/>
                    </div>
                    <label>Email address</label>
                    <input type="email" className="form-control form-box" aria-describedby="emailHelp" placeholder="Enter email"/>
                    <small id="emailHelp" className="form-text text-muted col-sm-8">
                        We'll never share your email with anyone else.
                    </small>
                </div><button type="submit" className="btn btn-primary button-footer">Submit</button>
                </form>
            </div>
        )
    }
}

class Faq extends Component {
    render() {
        return (
            <div className="form-align uber-color2">
                <h4 className="h9-font">
                    <br/>FAQ :
                </h4>
                <div className="uber-color2 description-txt">
                    <h6 className="question">
                        • Comment Überschutz protège mon enfant ?
                    </h6>
                    <h7>
                        Überschutz propose plusieurs manières de protéger son enfant. <br/>
                        Cela va de bloquer complètement Internet, jusqu'à bloquer uniquement certains sites tout en passant par d'autres moyens comme la restriction de sites ou la prévention.
                    </h7> <br/><br/>
                    <h6 className="question">
                        • En quoi Überschutz est différent d'un contrôle parental ?
                    </h6>
                    <h7>
                        (même réponse qu'en bas ?)
                    </h7> <br/><br/>
                    <h6 className="question">
                        • En quoi Überschutz protège mieux qu'un contrôle parental ?
                    </h6>
                    <h7>
                        Überschutz est mieux qu'un contrôle parental car il ne se contente pas uniquement de bloquer certains sites. <br/>
                        Überschutz est paramétré selon les envies et les règles que les parents veulent imposer à leurs enfants. A eux de choisir le meilleur moyen pour les protéger.
                    </h7> <br/><br/>
                    <h6 className="question">
                        • Qu'est-ce que les parents peuvent faire avec ce logiciel <h7 className="address">(bloquer, avertir, être prévenu ...)</h7> ?
                    </h6>
                    <h7>
                        Les parents peuvent protéger leurs enfants sur Internet de la manière dont ils l'entendent. <br/>
                        Si couper leur connexion lorsqu'ils dépassent les limites leurs paraît être la bonne solution, Überschutz le fera. <br/>
                        Si ils préfèrent jouer sur la patience de leurs enfants pour accéder à un site particulier, Überschutz le fera. <br/>
                        Si pour eux, l'apprentissage autonome d'Internet est la meilleure solution, Überschutz préviendra les enfants lorsqu'ils dépasseront les limites imposées par les parents, leurs laissant le choix de continuer ou non avec un pop-up. <br/>
                        A tout moment, les parents peuvent voir l'activité qu'ont leurs enfants sur Internet grâce au monitoring disponible avec le logiciel. Cela leur permettra de voir si l'enfant est aggressif ou agressé, s'il visite des sites peu recommandé ou si c'est un élément toxique sur Internet.
                    </h7> <br/><br/>
                    <h6 className="question">
                        • A quel prix pouvez-vous protéger votre enfant ?
                    </h6>
                    <h7>
                        Nous proposons plusieurs formules et plusieurs types de logiciels. <br/>
                        Si vous voulez en savoir plus, rendez-vous sur la page <h7 className="address">"Produits"</h7> ou envoyer nous un mail à <h7 className="address">uberschutz_2021@labeip.epitech.eu</h7> pour avoir des renseignements supervisés.
                    </h7> <br/><br/>
                    <h6 className="question">
                        • Comment paramétrer l'application <h7 className="address">(web et mobile)</h7> ?
                    </h6>
                    <h7>
                        Pour les 2 plateformes, rendez-vous sur la page <h7 className="address">"Paramètres"</h7>, cliquez sur le prénom de l'enfant que vous voulez protéger et cochez les options que vous voulez appliquer à sa navigation Internet. <br/>
                        Choisissez le type de restriction que vous voulez <h7 className="address">(bloquer, limiter, avertir).</h7>
                    </h7> <br/><br/>
                    <h6 className="question">
                        • Comment initialiser Überschutz sur les appareils de mes enfants ?
                    </h6>
                    <h7>
                        Lors de votre 1ere visite, allez sur la page <h7 className="address"> "Connexion"</h7> et remplissez les champs demandés dans <h7 className="address">"Inscription".</h7> <br/>
                        Sur l'appareil fourni lors de l'achat de Überschutz, un numéro de série est présent. C'est celui-ci qu'il faut renseigner dans la partie <h7 className="address">"Numéro de série".</h7> <br/>
                        Vous serez ensuite redirigés vers votre profil où vous pourrez ajouter et paramétrer les options que vous voulez attribuer à la navigation Internet de votre enfant. <br/>
                        Branchez votre boîtier et ils seront automatiquement associés à ses appareils.
                    </h7> <br/> <br/>
                </div>
            </div>
        )
    }
}


/* ------- Definir "non bloquant" et "non intrusif"
* ---------Comment paramétrer l'application / logiciel ?
* ---------Comment initialiser le controle parental sur mon appareil ?
* ---------Aborder question du prix (abonnement)
* ---------Qu'est ce que peuvent faire les parents avec logiciel (bloqué, ou avertir, tout en étant prévenus ...) par systeme d'option
* ----------Comment Üz protège mon enfant ?
* ---------En quoi Üz est différent d'un contrôle parental ?
* ---------En quoi Üz protége mieux qu'un contrôle parental ?*/


export { Contact, Form, Faq };