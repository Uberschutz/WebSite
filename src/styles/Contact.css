.h9-font {
    font-size: 120%;
    font-weight: bold;
}

.address {
    font-style: italic;
}

.form-align {
    align-items: center!important;
    align-content: center;
    justify-content: center;
    justify-items: center;
    text-align: center;
}

.form-box {
    width: 50%;
    margin-left: 25%;
}

.uber-color2 {
    background-color: #1d2124;
    color: white!important;
}

.description-txt {
    text-align: left!important;
    margin: 7px;
    font-size: 100%;
}

.title-bold {
    font-weight: bold;
    font-size: 120%;
}

.question {
    font-weight: bold;
    font-size: 100%;
}

.button-footer {
    margin-top: 15px;
    margin-bottom: 15px;
}

.icon-size {
    font-size: 20px;
}