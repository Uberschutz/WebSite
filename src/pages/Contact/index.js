import { connect } from 'react-redux'
import { bindActionCreators } from 'redux'

import * as actions from '../../actions'
import Contact from './Contact'

const mapStateToProps = state => ({
	base: state.base,
})

const mapDispatchToProps = dispatch => ({
	setNewsletter: bindActionCreators(actions.setNewsletter, dispatch)
})

const ContactContainer = connect(mapStateToProps, mapDispatchToProps)(Contact)

export default ContactContainer