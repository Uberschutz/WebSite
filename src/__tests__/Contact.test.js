import React from 'react';
import ReactDOM from 'react-dom';
import Contact from '../pages/Contact';

it('renders Contact without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Contact lang='fr'/>, div);
    expect(div).toMatchSnapshot();
    ReactDOM.unmountComponentAtNode(div);
});
