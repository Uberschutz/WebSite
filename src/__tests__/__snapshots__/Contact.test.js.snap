// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Contact without crashing 1`] = `
<div>
  <div
    class="form-align"
  >
    <div>
       
      <br />
      <h9
        class="h9-font"
      >
        Qui sommes nous ? Que faisons nous ?
      </h9>
       
      <br />
       
      <br />
      <div
        class="description-txt"
      >
        <h6
          class="title-bold"
        >
          • Überschutz :
        </h6>
        <h7>
          Nous sommes un groupe de 7 étudiants d'Epitech de la promo 2021.
          <br />
          Notre groupe a été créé en octobre 2018 (tout d'abord 3 personnes), lors de la Moonshot, puis s'est complété courant janvier 2019.
        </h7>
         
        <br />
         
        <br />
        <h6
          class="title-bold"
        >
          • Notre but :
        </h6>
        <h7>
          Nous voulons qu'Internet soit un lieu de partage, de détente et de recherche.
          <br />
          Beaucoup trop de personnes et d'enfants sont harcelés, arnaqués ou confrontés à du contenu inapproprié.
          <br />
          Ainsi, nous souhaitons guider les plus jeunes afin qu'ils apprennent à utiliser Internet de la meilleure manière possible pour eux selon l'éducation que leurs parents veulent leurs donner.
        </h7>
         
        <br />
         
        <br />
        <h6
          class="title-bold"
        >
          • Notre solution :
        </h6>
        <h7>
          Aujourd'hui, en tant que jeunes adultes, nous trouvons que certains contenus sont choquants sur Internet et que ce n'est pas assez sécurisé.
          <br />
          Donc qu'en est-il pour un enfant ?
          <br />
          C'est pour cela que nous proposons un logiciel capable de monitorer la navigation Internet.
          <br />
          De ce fait, les enfants pourront être avertis que leur recherche n'est pas adaptée pour eux selon leurs parents.
          <br />
          Afin de respecter la liberté de l'enfant, Überschutz n'est ni bloquant ni intrusif.
          <br />
          C'est à dire qu'en fonction du paramétrage que les parents choisiront, le logiciel n'interviendra qu'en tant que superviseur. Il laissera l'enfant naviguer mais se permettera d'afficher un message lorsque le contenu du site visité risque d'être inapproprié.
        </h7>
         
        <br />
        <br />
      </div>
    </div>
    <div
      class="form-align uber-color2"
    >
      <h4
        class="h9-font"
      >
        <br />
        FAQ :
      </h4>
      <div
        class="uber-color2 description-txt"
      >
        <h6
          class="question"
        >
          <i
            aria-label="icon: question-circle"
            class="anticon anticon-question-circle icon-size"
          >
            <svg
              aria-hidden="true"
              class=""
              data-icon="question-circle"
              fill="currentColor"
              focusable="false"
              height="1em"
              viewBox="64 64 896 896"
              width="1em"
            >
              <path
                d="M512 64C264.6 64 64 264.6 64 512s200.6 448 448 448 448-200.6 448-448S759.4 64 512 64zm0 820c-205.4 0-372-166.6-372-372s166.6-372 372-372 372 166.6 372 372-166.6 372-372 372z"
                fill="#1890ff"
              />
              <path
                d="M512 140c-205.4 0-372 166.6-372 372s166.6 372 372 372 372-166.6 372-372-166.6-372-372-372zm0 632c-22.1 0-40-17.9-40-40s17.9-40 40-40 40 17.9 40 40-17.9 40-40 40zm62.9-219.5a48.3 48.3 0 0 0-30.9 44.8V620c0 4.4-3.6 8-8 8h-48c-4.4 0-8-3.6-8-8v-21.5c0-23.1 6.7-45.9 19.9-64.9 12.9-18.6 30.9-32.8 52.1-40.9 34-13.1 56-41.6 56-72.7 0-44.1-43.1-80-96-80s-96 35.9-96 80v7.6c0 4.4-3.6 8-8 8h-48c-4.4 0-8-3.6-8-8V420c0-39.3 17.2-76 48.4-103.3C430.4 290.4 470 276 512 276s81.6 14.5 111.6 40.7C654.8 344 672 380.7 672 420c0 57.8-38.1 109.8-97.1 132.5z"
                fill="#e6f7ff"
              />
              <path
                d="M472 732a40 40 0 1 0 80 0 40 40 0 1 0-80 0zm151.6-415.3C593.6 290.5 554 276 512 276s-81.6 14.4-111.6 40.7C369.2 344 352 380.7 352 420v7.6c0 4.4 3.6 8 8 8h48c4.4 0 8-3.6 8-8V420c0-44.1 43.1-80 96-80s96 35.9 96 80c0 31.1-22 59.6-56 72.7-21.2 8.1-39.2 22.3-52.1 40.9-13.2 19-19.9 41.8-19.9 64.9V620c0 4.4 3.6 8 8 8h48c4.4 0 8-3.6 8-8v-22.7a48.3 48.3 0 0 1 30.9-44.8c59-22.7 97.1-74.7 97.1-132.5 0-39.3-17.2-76-48.4-103.3z"
                fill="#1890ff"
              />
            </svg>
          </i>
           
          Comment Überschutz protège votre enfant ?
        </h6>
        <h7>
          Überschutz propose plusieurs manières de protéger les enfants.
          <br />
          Cela va de bloquer complètement Internet, jusqu'à bloquer uniquement certains sites tout en passant par d'autres moyens comme la restriction de sites ou la prévention.
        </h7>
         
        <br />
        <br />
        <h6
          class="question"
        >
          <i
            aria-label="icon: question-circle"
            class="anticon anticon-question-circle icon-size"
          >
            <svg
              aria-hidden="true"
              class=""
              data-icon="question-circle"
              fill="currentColor"
              focusable="false"
              height="1em"
              viewBox="64 64 896 896"
              width="1em"
            >
              <path
                d="M512 64C264.6 64 64 264.6 64 512s200.6 448 448 448 448-200.6 448-448S759.4 64 512 64zm0 820c-205.4 0-372-166.6-372-372s166.6-372 372-372 372 166.6 372 372-166.6 372-372 372z"
                fill="#1890ff"
              />
              <path
                d="M512 140c-205.4 0-372 166.6-372 372s166.6 372 372 372 372-166.6 372-372-166.6-372-372-372zm0 632c-22.1 0-40-17.9-40-40s17.9-40 40-40 40 17.9 40 40-17.9 40-40 40zm62.9-219.5a48.3 48.3 0 0 0-30.9 44.8V620c0 4.4-3.6 8-8 8h-48c-4.4 0-8-3.6-8-8v-21.5c0-23.1 6.7-45.9 19.9-64.9 12.9-18.6 30.9-32.8 52.1-40.9 34-13.1 56-41.6 56-72.7 0-44.1-43.1-80-96-80s-96 35.9-96 80v7.6c0 4.4-3.6 8-8 8h-48c-4.4 0-8-3.6-8-8V420c0-39.3 17.2-76 48.4-103.3C430.4 290.4 470 276 512 276s81.6 14.5 111.6 40.7C654.8 344 672 380.7 672 420c0 57.8-38.1 109.8-97.1 132.5z"
                fill="#e6f7ff"
              />
              <path
                d="M472 732a40 40 0 1 0 80 0 40 40 0 1 0-80 0zm151.6-415.3C593.6 290.5 554 276 512 276s-81.6 14.4-111.6 40.7C369.2 344 352 380.7 352 420v7.6c0 4.4 3.6 8 8 8h48c4.4 0 8-3.6 8-8V420c0-44.1 43.1-80 96-80s96 35.9 96 80c0 31.1-22 59.6-56 72.7-21.2 8.1-39.2 22.3-52.1 40.9-13.2 19-19.9 41.8-19.9 64.9V620c0 4.4 3.6 8 8 8h48c4.4 0 8-3.6 8-8v-22.7a48.3 48.3 0 0 1 30.9-44.8c59-22.7 97.1-74.7 97.1-132.5 0-39.3-17.2-76-48.4-103.3z"
                fill="#1890ff"
              />
            </svg>
          </i>
           
          En quoi Überschutz est différent d'un contrôle parental ? Et en quoi protège-t-il mieux ?
        </h6>
        <h7>
          Überschutz est mieux qu'un contrôle parental car il ne se contente pas uniquement d'être un obstacle à la navigation des enfants.
          <br />
          Il s'adapte selon les dangers de chaque génération, intervient uniquement dans certains cas de figure réellement néfastes et laisse un certain libre arbitre avec des limites qui seront connues de l'utilisateur.
          <br />
          Überschutz est paramétré selon les envies et les règles que les parents veulent inculquer à leurs enfants. A eux de choisir le meilleur moyen de les protéger.
          <br />
          En d'autres termes, Überschutz est mieux qu'un contrôle parental car le logiciel est en mesure de comprendre l'enfant et non de lui faire barrage. Ici il ne s'agit plus de surveillance mais d'accompagnement.
        </h7>
         
        <br />
        <br />
        <h6
          class="question"
        >
          <i
            aria-label="icon: question-circle"
            class="anticon anticon-question-circle icon-size"
          >
            <svg
              aria-hidden="true"
              class=""
              data-icon="question-circle"
              fill="currentColor"
              focusable="false"
              height="1em"
              viewBox="64 64 896 896"
              width="1em"
            >
              <path
                d="M512 64C264.6 64 64 264.6 64 512s200.6 448 448 448 448-200.6 448-448S759.4 64 512 64zm0 820c-205.4 0-372-166.6-372-372s166.6-372 372-372 372 166.6 372 372-166.6 372-372 372z"
                fill="#1890ff"
              />
              <path
                d="M512 140c-205.4 0-372 166.6-372 372s166.6 372 372 372 372-166.6 372-372-166.6-372-372-372zm0 632c-22.1 0-40-17.9-40-40s17.9-40 40-40 40 17.9 40 40-17.9 40-40 40zm62.9-219.5a48.3 48.3 0 0 0-30.9 44.8V620c0 4.4-3.6 8-8 8h-48c-4.4 0-8-3.6-8-8v-21.5c0-23.1 6.7-45.9 19.9-64.9 12.9-18.6 30.9-32.8 52.1-40.9 34-13.1 56-41.6 56-72.7 0-44.1-43.1-80-96-80s-96 35.9-96 80v7.6c0 4.4-3.6 8-8 8h-48c-4.4 0-8-3.6-8-8V420c0-39.3 17.2-76 48.4-103.3C430.4 290.4 470 276 512 276s81.6 14.5 111.6 40.7C654.8 344 672 380.7 672 420c0 57.8-38.1 109.8-97.1 132.5z"
                fill="#e6f7ff"
              />
              <path
                d="M472 732a40 40 0 1 0 80 0 40 40 0 1 0-80 0zm151.6-415.3C593.6 290.5 554 276 512 276s-81.6 14.4-111.6 40.7C369.2 344 352 380.7 352 420v7.6c0 4.4 3.6 8 8 8h48c4.4 0 8-3.6 8-8V420c0-44.1 43.1-80 96-80s96 35.9 96 80c0 31.1-22 59.6-56 72.7-21.2 8.1-39.2 22.3-52.1 40.9-13.2 19-19.9 41.8-19.9 64.9V620c0 4.4 3.6 8 8 8h48c4.4 0 8-3.6 8-8v-22.7a48.3 48.3 0 0 1 30.9-44.8c59-22.7 97.1-74.7 97.1-132.5 0-39.3-17.2-76-48.4-103.3z"
                fill="#1890ff"
              />
            </svg>
          </i>
           
          Que pouvez-vous faire avec ce logiciel 
          <h7
            class="address"
          >
            (bloquer, avertir, être prévenu ...) 
          </h7>
           ?
        </h6>
        <h7>
          Les parents peuvent protéger leurs enfants sur Internet de la manière dont ils l'entendent.
          <br />
          Si couper leur connexion lorsqu'ils dépassent les limites leurs paraît être la bonne solution, Überschutz le fera.
          <br />
          Si ils préfèrent jouer sur la patience de leurs enfants pour accéder à un site particulier, Überschutz le fera.
          <br />
          Si pour eux, l'apprentissage autonome d'Internet est la meilleure solution, Überschutz préviendra les enfants lorsqu'ils dépasseront les limites imposées par les parents, leurs laissant le choix de continuer ou non avec un pop-up.
          <br />
          A tout moment, les parents peuvent voir l'activité qu'ont leurs enfants sur Internet grâce au monitoring disponible avec le logiciel. Cela leur permettra de voir si l'enfant est aggressif ou agressé, s'il visite des sites peu recommandé ou si c'est un élément toxique sur Internet.
        </h7>
         
        <br />
        <br />
        <h6
          class="question"
        >
          <i
            aria-label="icon: question-circle"
            class="anticon anticon-question-circle icon-size"
          >
            <svg
              aria-hidden="true"
              class=""
              data-icon="question-circle"
              fill="currentColor"
              focusable="false"
              height="1em"
              viewBox="64 64 896 896"
              width="1em"
            >
              <path
                d="M512 64C264.6 64 64 264.6 64 512s200.6 448 448 448 448-200.6 448-448S759.4 64 512 64zm0 820c-205.4 0-372-166.6-372-372s166.6-372 372-372 372 166.6 372 372-166.6 372-372 372z"
                fill="#1890ff"
              />
              <path
                d="M512 140c-205.4 0-372 166.6-372 372s166.6 372 372 372 372-166.6 372-372-166.6-372-372-372zm0 632c-22.1 0-40-17.9-40-40s17.9-40 40-40 40 17.9 40 40-17.9 40-40 40zm62.9-219.5a48.3 48.3 0 0 0-30.9 44.8V620c0 4.4-3.6 8-8 8h-48c-4.4 0-8-3.6-8-8v-21.5c0-23.1 6.7-45.9 19.9-64.9 12.9-18.6 30.9-32.8 52.1-40.9 34-13.1 56-41.6 56-72.7 0-44.1-43.1-80-96-80s-96 35.9-96 80v7.6c0 4.4-3.6 8-8 8h-48c-4.4 0-8-3.6-8-8V420c0-39.3 17.2-76 48.4-103.3C430.4 290.4 470 276 512 276s81.6 14.5 111.6 40.7C654.8 344 672 380.7 672 420c0 57.8-38.1 109.8-97.1 132.5z"
                fill="#e6f7ff"
              />
              <path
                d="M472 732a40 40 0 1 0 80 0 40 40 0 1 0-80 0zm151.6-415.3C593.6 290.5 554 276 512 276s-81.6 14.4-111.6 40.7C369.2 344 352 380.7 352 420v7.6c0 4.4 3.6 8 8 8h48c4.4 0 8-3.6 8-8V420c0-44.1 43.1-80 96-80s96 35.9 96 80c0 31.1-22 59.6-56 72.7-21.2 8.1-39.2 22.3-52.1 40.9-13.2 19-19.9 41.8-19.9 64.9V620c0 4.4 3.6 8 8 8h48c4.4 0 8-3.6 8-8v-22.7a48.3 48.3 0 0 1 30.9-44.8c59-22.7 97.1-74.7 97.1-132.5 0-39.3-17.2-76-48.4-103.3z"
                fill="#1890ff"
              />
            </svg>
          </i>
           
          A quel prix pouvez-vous protéger votre enfant ?
        </h6>
        <h7>
          Nous proposons plusieurs formules et plusieurs types de logiciels.
          <br />
          Si vous voulez en savoir plus, rendez-vous sur la page 
          <h7
            class="address"
          >
            "Produits"
          </h7>
           ou envoyer nous un mail à 
          <h7
            class="address"
          >
            uberschutz_2021@labeip.epitech.eu
          </h7>
           pour avoir des renseignements supervisés.
        </h7>
         
        <br />
        <br />
        <h6
          class="question"
        >
          <i
            aria-label="icon: question-circle"
            class="anticon anticon-question-circle icon-size"
          >
            <svg
              aria-hidden="true"
              class=""
              data-icon="question-circle"
              fill="currentColor"
              focusable="false"
              height="1em"
              viewBox="64 64 896 896"
              width="1em"
            >
              <path
                d="M512 64C264.6 64 64 264.6 64 512s200.6 448 448 448 448-200.6 448-448S759.4 64 512 64zm0 820c-205.4 0-372-166.6-372-372s166.6-372 372-372 372 166.6 372 372-166.6 372-372 372z"
                fill="#1890ff"
              />
              <path
                d="M512 140c-205.4 0-372 166.6-372 372s166.6 372 372 372 372-166.6 372-372-166.6-372-372-372zm0 632c-22.1 0-40-17.9-40-40s17.9-40 40-40 40 17.9 40 40-17.9 40-40 40zm62.9-219.5a48.3 48.3 0 0 0-30.9 44.8V620c0 4.4-3.6 8-8 8h-48c-4.4 0-8-3.6-8-8v-21.5c0-23.1 6.7-45.9 19.9-64.9 12.9-18.6 30.9-32.8 52.1-40.9 34-13.1 56-41.6 56-72.7 0-44.1-43.1-80-96-80s-96 35.9-96 80v7.6c0 4.4-3.6 8-8 8h-48c-4.4 0-8-3.6-8-8V420c0-39.3 17.2-76 48.4-103.3C430.4 290.4 470 276 512 276s81.6 14.5 111.6 40.7C654.8 344 672 380.7 672 420c0 57.8-38.1 109.8-97.1 132.5z"
                fill="#e6f7ff"
              />
              <path
                d="M472 732a40 40 0 1 0 80 0 40 40 0 1 0-80 0zm151.6-415.3C593.6 290.5 554 276 512 276s-81.6 14.4-111.6 40.7C369.2 344 352 380.7 352 420v7.6c0 4.4 3.6 8 8 8h48c4.4 0 8-3.6 8-8V420c0-44.1 43.1-80 96-80s96 35.9 96 80c0 31.1-22 59.6-56 72.7-21.2 8.1-39.2 22.3-52.1 40.9-13.2 19-19.9 41.8-19.9 64.9V620c0 4.4 3.6 8 8 8h48c4.4 0 8-3.6 8-8v-22.7a48.3 48.3 0 0 1 30.9-44.8c59-22.7 97.1-74.7 97.1-132.5 0-39.3-17.2-76-48.4-103.3z"
                fill="#1890ff"
              />
            </svg>
          </i>
           
          Comment paramétrer l'application 
          <h7
            class="address"
          >
            (web et mobile)
          </h7>
           ?
        </h6>
        <h7>
          Pour les 2 plateformes, rendez-vous sur la page 
          <h7
            class="address"
          >
            "Paramètres"
          </h7>
          , cliquez sur le prénom de l'enfant que vous voulez protéger et cochez les options que vous voulez appliquer à sa navigation Internet.
          <br />
          Choisissez le type de restriction que vous voulez 
          <h7
            class="address"
          >
            (bloquer, limiter, avertir).
          </h7>
        </h7>
         
        <br />
        <br />
        <h6
          class="question"
        >
          <i
            aria-label="icon: question-circle"
            class="anticon anticon-question-circle icon-size"
          >
            <svg
              aria-hidden="true"
              class=""
              data-icon="question-circle"
              fill="currentColor"
              focusable="false"
              height="1em"
              viewBox="64 64 896 896"
              width="1em"
            >
              <path
                d="M512 64C264.6 64 64 264.6 64 512s200.6 448 448 448 448-200.6 448-448S759.4 64 512 64zm0 820c-205.4 0-372-166.6-372-372s166.6-372 372-372 372 166.6 372 372-166.6 372-372 372z"
                fill="#1890ff"
              />
              <path
                d="M512 140c-205.4 0-372 166.6-372 372s166.6 372 372 372 372-166.6 372-372-166.6-372-372-372zm0 632c-22.1 0-40-17.9-40-40s17.9-40 40-40 40 17.9 40 40-17.9 40-40 40zm62.9-219.5a48.3 48.3 0 0 0-30.9 44.8V620c0 4.4-3.6 8-8 8h-48c-4.4 0-8-3.6-8-8v-21.5c0-23.1 6.7-45.9 19.9-64.9 12.9-18.6 30.9-32.8 52.1-40.9 34-13.1 56-41.6 56-72.7 0-44.1-43.1-80-96-80s-96 35.9-96 80v7.6c0 4.4-3.6 8-8 8h-48c-4.4 0-8-3.6-8-8V420c0-39.3 17.2-76 48.4-103.3C430.4 290.4 470 276 512 276s81.6 14.5 111.6 40.7C654.8 344 672 380.7 672 420c0 57.8-38.1 109.8-97.1 132.5z"
                fill="#e6f7ff"
              />
              <path
                d="M472 732a40 40 0 1 0 80 0 40 40 0 1 0-80 0zm151.6-415.3C593.6 290.5 554 276 512 276s-81.6 14.4-111.6 40.7C369.2 344 352 380.7 352 420v7.6c0 4.4 3.6 8 8 8h48c4.4 0 8-3.6 8-8V420c0-44.1 43.1-80 96-80s96 35.9 96 80c0 31.1-22 59.6-56 72.7-21.2 8.1-39.2 22.3-52.1 40.9-13.2 19-19.9 41.8-19.9 64.9V620c0 4.4 3.6 8 8 8h48c4.4 0 8-3.6 8-8v-22.7a48.3 48.3 0 0 1 30.9-44.8c59-22.7 97.1-74.7 97.1-132.5 0-39.3-17.2-76-48.4-103.3z"
                fill="#1890ff"
              />
            </svg>
          </i>
           
          Comment initialiser Überschutz sur les appareils de vos enfants ?
        </h6>
        <h7>
          Lors de votre 1ere visite, allez sur la page 
          <h7
            class="address"
          >
            "Connection"
          </h7>
           et remplissez les champs demandés dans 
          <h7
            class="address"
          >
            "Inscription"
            .
          </h7>
           
          <br />
          Sur l'appareil fourni lors de l'achat de Überschutz, un numéro de série est présent. C'est celui-ci qu'il faut renseigner dans la partie 
          <h7
            class="address"
          >
            "Numéro de série".
          </h7>
           
          <br />
          Vous serez ensuite redirigés vers votre profil où vous pourrez ajouter et paramétrer les options que vous voulez attribuer à la navigation Internet de votre enfant.
          <br />
          Branchez votre boîtier et ils seront automatiquement associés à ses appareils.
        </h7>
         
        <br />
         
        <br />
      </div>
    </div>
    <div>
      <br />
      <h9
        class="h9-font"
      >
        Vous êtes intéressés par notre solution ?
      </h9>
      <br />
      <h11>
        N'hésitez pas à vous tenir informés en remplissant ce formulaire !
      </h11>
      <br />
      <h11>
        Ou en nous contactant à l'adresse suivante : 
        <h11
          class="address"
        >
          uberschutz_2021@labeip.epitech.eu
        </h11>
      </h11>
      <form>
        <br />
        <div
          class="form-group"
        >
          <div
            class="form-group form-align"
          >
            <label>
              Nom ou Prénom
            </label>
            <input
              class="form-control form-box"
              placeholder="Nom ou Prénom"
              type="text"
            />
          </div>
          <label>
            Adresse email
          </label>
          <input
            aria-describedby="emailHelp"
            class="form-control form-box"
            placeholder="Adresse email"
            type="email"
          />
          <small
            class="form-text text-muted col-sm-9"
            id="emailHelp"
          >
            Nous ne partagerons jamais votre adresse email avec qui que ce soit.
          </small>
        </div>
        <button
          class="btn btn-primary button-footer"
          type="submit"
        >
          Soumettre
        </button>
      </form>
    </div>
  </div>
</div>
`;
